### ISSUE TITLE
[Error] json: cannot unmarshal object into Go struct field Result.parameters of type string

### ISSUE THREAD

**Issue by darrimajou** [yoichiro/dialogflow-query-checker#27]
Hi,

I use your test framework with my Dialogflow weather POC, but I got this issue : 

```
./dialogflow-query-checker run test.yml -d
dialogflow-query-checker version 1.11.0
The configuration file loaded: test.yml
The debug mode is on
The count of retrying: 0
Running query tests for Dialogflow.
[Start] Query: météo à Bordeaux (weather)
<Request body> {
  "contexts": [
    "weather"
  ],
  "lang": "fr",
  "originalRequest": {
    "source": "google",
    "data": {
      "user": {
        "locale": "fr-FR"
      }
    }
  },
  "query": "météo à Bordeaux",
  "event": {},
  "sessionId": "da2257ab-e844-4264-873b-7f21171ab138"
}
<Response Status> 200 OK
<Response body> {
  "id": "48cc05aa-b092-4980-b0a5-a41b5f843556",
  "timestamp": "2018-03-05T13:30:39.729Z",
  "lang": "fr",
  "result": {
    "source": "agent",
    "resolvedQuery": "météo à Bordeaux",
    "action": "weather",
    "actionIncomplete": false,
    "parameters": {
      "address": {
        "city": "Bordeaux"
      },
      "date-time": ""
    },
    "contexts": [
      {
        "name": "_actions_on_google_",
        "parameters": {},
        "lifespan": 100
      },
      {
        "name": "weather",
        "parameters": {
          "date-time.original": "",
          "address": {
            "city": "Bordeaux",
            "city.original": "Bordeaux"
          },
          "date-time": "",
          "address.original": "Bordeaux"
        },
        "lifespan": 5
      }
    ],
    "metadata": {
      "intentId": "f1b75ecb-a35f-4a26-88fb-5a8049b92b02",
      "webhookUsed": "true",
      "webhookForSlotFillingUsed": "true",
      "webhookResponseTime": 1050,
      "intentName": "weather"
    },
    "fulfillment": {
      "speech": "légère pluie à Bordeaux il fera au minimum 9 degrès et au maximum 16 degrès",
      "messages": [
        {
          "type": 0,
          "speech": "légère pluie à Bordeaux il fera au minimum 9 degrès et au maximum 16 degrès"
        }
      ],
      "data": {
        "google": {
          "expect_user_response": true,
          "no_input_prompts": [],
          "rich_response": {
            "items": [
              {
                "simple_response": {
                  "text_to_speech": "légère pluie à Bordeaux il fera au minimum 9 degrès et au maximum 16 degrès"
                }
              },
              {
                "basic_card": {
                  "title": "légère pluie à Bordeaux",
                  "formatted_text": "Température maximale : 16°C   \nTempérature minimale : 9°C",
                  "buttons": [],
                  "image_display_options": "CROPPED"
                }
              }
            ],
            "suggestions": []
          },
          "user_storage": "{\"data\":{}}"
        }
      }
    },
    "score": 1.0
  },
  "status": {
    "code": 200,
    "errorType": "success",
    "webhookTimedOut": false
  },
  "sessionId": "da2257ab-e844-4264-873b-7f21171ab138"
}

[Error] json: cannot unmarshal object into Go struct field Result.parameters of type string
```

May be you expect that parameters are string, but they can be objects.

```
"parameters": {
      "address": {
        "city": "Bordeaux"
      },
      "date-time": ""
    },
```

**Comment by yoichiro** [comment]
> May be you expect that parameters are string, but they can be objects.

@darrimajou Yes, I understand that the value of parameters can be string only, but I just know that the type can also become an object. I intend to handle this issue as bug, therefore I think that this issue take a priority than others.

**Comment by yoichiro** [comment]
@darrimajou I have just released a new version 1.14.0. In this version, specifying an object supported to assert each structured parameter. Could you test the version for your test-cases?

**Comment by yoichiro** [comment]
@darrimajou I close this issue. If you find some problem, please report it to me as a new issue. Thank you for this reporting!
